server/core_authenticate.go

```diff
@@ -384,6 +384,76 @@ func AuthenticateFacebook(ctx context.Context, logger *zap.Logger, db *sql.DB, c
 	return userID, username, true, nil
 }
 
+func extractInstantGameID(signedPlayerInfo string, appSecret string) (facebookInstantGameID string, err error) {
```

Comment:
If this is the function that will handle validating the token and potentially requesting the user's profile (and friends) from Facebook services then it should live in the `social` package.

Reply:
It doesn't request anything from FB, only validates the token and extracts the Facebook Instant Game player ID from it. 

Moved to `social` nonetheless.